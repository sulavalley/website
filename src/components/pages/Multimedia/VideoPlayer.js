import React from 'react';
import { Embed } from 'semantic-ui-react';
import '../../../css/pages/Multimedia.css';
import logo from '../../../assets/images/logo.png';

const VideoPlayer = props => (
  <div className="video-player">
    <Embed
      url={props.url}
      placeholder={logo}
      iframe={{
        allowFullScreen: true
      }}
    />
  </div>
);

export default VideoPlayer;
