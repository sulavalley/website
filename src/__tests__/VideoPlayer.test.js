import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import VideoPlayer from '../components/VideoPlayer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<VideoPlayer />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders without crashing', () => {
  shallow(<VideoPlayer />);
});

test('renders correctly', () => {
  const tree = renderer.create(<VideoPlayer />).toJSON();
  expect(tree).toMatchSnapshot();
});
