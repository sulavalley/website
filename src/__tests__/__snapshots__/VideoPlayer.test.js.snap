// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="video-player"
>
  <div
    className="ui text fluid justified container"
  >
    En esta secci√≥n encontraras videos sobre las actividades realizadas por la MRTVS.
  </div>
  <div
    className="ui divider"
  />
  <div
    className="ui embed"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="video play icon"
    />
    <img
      className="placeholder"
      src="logo.png"
    />
  </div>
</div>
`;
